.npmignore

```diff
@@ -0,0 +1,2 @@
+test
+*.*
```

Comment:
@kellyselden actually it seems this was excluding *every* matching file, not only those in the root folder. So v0.5.0 was basically an empty package. :(
Wasn't aware of this either, unfortunately. Fixed it by using `/*.*` which worked as expected. Released 0.5.1.